src/modules/main/navbar/hooks/useWindowSize.ts

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react';
+
+export interface Size {
+  width: number;
+  height: number;
+}
+
+export const useWindowSize = () => {
```

Comment:
implementing window size "detection" by js makes application performance really bad. Just check how many times the event is triggered when you change browser size. This should be debounced with RxJs.

Also implementing features like this creating big issues. We using JS and CSS for our layout things. Please remove this hook. Button replacement can be done by pure css.

Also, SSR technique will fail here - because of js usage - content will be displayed incorrectly for some devices and SSR thing will be implemented later for some views